### PR TITLE
[CELEBORN-1684] Fix ambiguous client jar expression of document

### PR DESCRIPTION
**Pull request by zaynt4606**
### What changes were proposed in this pull request?
When users deploy using the release binary as outlined in the documentation, the instructions for copying the client JAR can be unclear.

### Why are the changes needed?
No

### Does this PR introduce _any_ user-facing change?
No

### How was this patch tested?
image

### CODE REVIEW

**Review by turboFei — APPROVED** [review]

File: docs/deploy.md [hunk 1]
```diff
@@ -155,7 +155,10 @@ WorkerRef: null
 ```
 
 ## Deploy Spark client
-Copy `$CELEBORN_HOME/spark/*.jar` to `$SPARK_HOME/jars/`.
+Celeborn release binary contains clients for Spark 2.x and Spark 3.x, copy the corresponding client jar into Spark's
+`jars/` directory:
+
+Copy `$CELEBORN_HOME/spark/<Celeborn Client Jar>` to `$SPARK_HOME/jars/`.
```
Comment (SteNicholas) [inline docs/deploy.md:161]: The `<Celeborn Client Jar>` is also ambiguous.

**Comment by SteNicholas** [comment]
@zaynt4606, you should also update the changes in `README.md`.

File: docs/deploy.md (hunk 1 above)
Comment (zaynt4606) [reply docs/deploy.md:161]: What about <celeborn-client-spark-{version} Jar>

Comment (SteNicholas) [reply docs/deploy.md:161]: LGTM.

**Comment by zaynt4606** [comment]
README changed too.

File: docs/deploy.md (hunk 1 above)
Comment (SteNicholas) [reply docs/deploy.md]: @zaynt4606, `celeborn-client-spark-<spark.major.version>-shaded_<scala.binary.version>-<celeborn.version>.jar`, WDYT?

**Review by SteNicholas — APPROVED** [review]

**Comment by SteNicholas** [comment]
cc @RexXiong.

**Review by RexXiong — APPROVED** [review]
LGTM, thanks!

**Review by cxzl25 — APPROVED** [review]

**Comment by SteNicholas** [comment]
Merged to main(v0.6.0).

**Pull request closed without merging**